### PR TITLE
Fix unit tests & build w/o ZeroEQ

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

**Comment by bilgili** [comment]
I am checking the zeq issue

**Comment by tribal-tec** [comment]
retest this please

**Comment by tribal-tec** [comment]
updated. ready to merge?

File: tests/Data/dataSource.cpp [hunk 1]
```diff
@@ -75,7 +75,7 @@ void _testDataSource( const std::string& uriStr )
 BOOST_AUTO_TEST_CASE( memoryDataSource )
 {
     std::stringstream volumeName;
-    volumeName << "mem:///#" << VOXEL_SIZE_X << "," << VOXEL_SIZE_Y << ","
+    volumeName << "mem://#" << VOXEL_SIZE_X << "," << VOXEL_SIZE_Y << ","
```
Comment (hernando) [inline tests/Data/dataSource.cpp]: "men:///#" is valid , but even "mem:#whatever" is a valid URI. The // separator is needed when you are going to specify an authority, to distinguish it from an absolute path (a single /).

**Comment by hernando** [comment]
+1 if you don't want to use HBPVIS/Servus#10.

**Pull request merged**